airbyte-integrations/connectors/destination-elasticsearch/src/test-integration/java/io/airbyte/integrations/destination/elasticsearch/SshElasticsearchDestinationAcceptanceTest.java

```diff
@@ -0,0 +1,127 @@
+package io.airbyte.integrations.destination.elasticsearch;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.integrations.base.ssh.SshBastionContainer;
+import io.airbyte.integrations.base.ssh.SshTunnel;
+import io.airbyte.integrations.standardtest.destination.DestinationAcceptanceTest;
+import io.airbyte.integrations.standardtest.destination.comparator.AdvancedTestDataComparator;
+import io.airbyte.integrations.standardtest.destination.comparator.TestDataComparator;
+import java.util.List;
+import org.testcontainers.containers.Network;
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+
+public abstract class SshElasticsearchDestinationAcceptanceTest  extends DestinationAcceptanceTest {
```

Comment:
Overall looks great, just curious was there a particular reason this doesn't extend from `ElasticsearchDestinationAcceptanceTest` as some of these Overrides are already in that sub-class: `implementsNamespaces`, `supportsNormalization`, `supportBasicDataTypeTest`, etc

Reply:
Good catch. I did this initially, because I was trying to use a different version of Elastic container, but in the current implementation of this test it makes sense to inherit from `ElasticsearchDestinationAcceptanceTest`